bigtable/google/cloud/bigtable/row.py

```diff
@@ -412,9 +427,16 @@ def commit(self):
             row_key=self._row_key,
             mutations=mutations_list,
         )
-        # We expect a `google.protobuf.empty_pb2.Empty`
-        client = self._table._instance._client
-        client._data_stub.MutateRow(request_pb)
+
+        retry_commit = functools.partial(
```

Comment:
It makes more sense for this to just be called `commit` not `retry_commit` as it doesn't in itself retry. Then, the application of `retry_` below reads `retry_(commit)()`.